eventastic_outbox_postgres/src/outbox.rs

```diff
@@ -101,15 +106,22 @@ where
 }
 
 #[async_trait]
-impl<T, E> TransactionOutboxExt<T, E::Error> for PostgresTransaction<'_, TableOutbox<E>, E>
+impl<T, E> TransactionOutboxExt<T::SideEffect, E::Error>
+    for PostgresTransaction<'_, T, TableOutbox<E>, E>
 where
-    T: SideEffect + Pickle + Send + 'static,
-    T::SideEffectId: Clone + Send + 'static,
-    for<'sql> T::SideEffectId:
+    T: Aggregate<AggregateId = Uuid> + Send + Sync + Pickle + 'static,
+    T::SideEffect: SideEffect + Pickle + Clone + Send + Sync + 'static,
+    <T::SideEffect as SideEffect>::SideEffectId: Clone + Send + 'static,
+    T::SideEffect: SideEffect<SideEffectId = Uuid> + Pickle + Send + Sync,
```

Comment:
Think these are duplicates.

```suggestion
    T::SideEffect: SideEffect<SideEffectId = Uuid> + Pickle + Send + Sync,
```